tests/unit/IonTextReaderTest.js

```diff
@@ -226,6 +226,19 @@ define(
 
 
         }
+
+        function testDeepValue(ionToRead, expectedOutput = ionToRead) {
+            var ionReader = ion.makeReader(ionToRead);
+            assert.equal(ionReader.deepValue(), expectedOutput);
```

Comment:
Again, to have this work the way it is asserted below, wouldn't you have to `next` first?